src/contexts/ToastContext.tsx

```diff
@@ -0,0 +1,51 @@
+'use client'
+import { createContext, useState, useContext, PropsWithChildren } from 'react'
+
+export enum ToastStatus {
+  SUCCESS,
+  INVALID,
+  HIDDEN,
+}
+
+export type ToastType = PropsWithChildren & {
+  message: string
+  toastStatus: ToastStatus
+  setToast: (toastStatus: ToastStatus, message: string) => void
+}
+
+export const ToastContext = createContext<ToastType | null>(null)
+
+export function ToastContextProvider({
+  children,
+}: {
+  children: React.ReactNode
+}) {
+  const [message, setMessage] = useState('')
+  const [toastStatus, setToastStatus] = useState<ToastStatus>(
```

Comment:
**issue:** This toaster is missing queueing - toasts should stack. Currently they will overwrite themselves